README.md

```diff
@@ -27,6 +27,7 @@ This extension requires that Node.js and npm be installed on the build agent. Th
 | dockerfilePath            | The path to the Dockerfile corresponding to the `dockerImageName`                                                                                                                                                   | yes, if container image test | none          | string                                                                            |
 | targetFile                | Applicable to application type tests ony. The path to the manifest file to be used by Snyk. Should only be provided if non-standard.                                                                                | no                           | none          | string                                                                            |
 | severityThreshold         | The severity-threshold to use when testing. By default, issues of all severity types will be found.                                                                                                                 | no                           | "low"         | string: "low" or "medium" or "high" or "critical"                                 |
+| failOnThreshold           | The severity-threshold to use when determining if the build should fail. Combine with `failOnIssues` to get fine grained control over reporting and build failure behaviour. For example, with `failOnIssues` set to true and `failOnThreshold` to `critical`, all issues would be reported on but only critical issues would cause a build failure                                                                                                            | no                           | "low"         | string: "low" or "medium" or "high" or "critical"                                 |
```

Comment:
Could you please mention that `severityThreshold` parameter is responsible for reporting? We should avoid people expecting that  `failOnThreshold` is also responsible for reporting.